query/src/test/java/org/infinispan/query/mapper/ProgrammaticSearchMappingProviderTest.java

```diff
@@ -1,7 +1,5 @@
 package org.infinispan.query.mapper;
 
-import org.hibernate.search.mapper.pojo.mapping.definition.annotation.FullTextField;
-import org.hibernate.search.mapper.pojo.mapping.definition.annotation.Indexed;
 import org.hibernate.search.mapper.pojo.mapping.definition.programmatic.ProgrammaticMappingConfigurationContext;
 import org.hibernate.search.mapper.pojo.mapping.definition.programmatic.TypeMappingStep;
```

Comment:
btw, I don't mind it 😃,
but is it OK to leak the Search classes into Infinispan ? meaning it seems that the Infinispan's `ProgrammaticSearchMappingProvider`  interface exposes the Search classes to the Infinispan users ... Maybe there should be some kind of wrapper around that?